web/template/edit_document.gohtml

```diff
@@ -0,0 +1,76 @@
+{{ template "page" . }}
+
+{{ define "title" }}{{ if .Error.Any }}Error: {{ end }}Edit Document{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ template "error-summary" .Error }}
+
+            {{ if .Success }}
+                <meta data-app-reload="page" />
+                {{ template "success-banner" "You have successfully published a document." }}
+            {{ else if .SaveAndExit }}
+                <meta data-app-reload="saveAndExit" />
+                {{ template "success-banner" "You have successfully saved the document." }}
+            {{ else if .PreviewDraft }}
+                <meta data-app-reload="previewDraft" id="{{ sirius (printf "/lpa-api/v1/documents/%s/download" .DownloadUUID) }}"/>
```

Comment:
Rather than depending on JavaScript to open the preview, I think it would be a more resilient to make this a link, and then make the JavaScript auto-open it. So something like:
```html
<a href="{{ sirius (printf "/lpa-api/v1/documents/%s/download" .DownloadUUID) }}" target="_blank" data-module="app-auto-click">Download preview document</a>
```
And then JS like:
```js
document.querySelectorAll('[data-module~="app-auto-click"]').forEach($link => {
  $link.click();
  $link.hidden = true;
});
```